packages/shared/src/modules/notifications/libs/types/notification-bulk-mark-as-read-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type NotificationBulkMarkAsReadRequestDto = {
+	notificationIds: number[];
```

Comment:
we don't need to pass anything to the backend, this request means to mark all notifications as read
```suggestion
```